ta/ta.mk

```diff
@@ -41,6 +41,7 @@ base-prefix :=
 incdirs-host := $(filter-out lib/libutils%, $(incdirs$(sm)))
 incfiles-extra-host := lib/libutils/ext/include/compiler.h
 incfiles-extra-host += lib/libutils/ext/include/util.h
+incfiles-extra-host += lib/libutils/ext/include/types_ext.h
```

Comment:
Having `stdint.h` increases the risk that a TA includes the wrong stdint.h file, i.e. the one of the compiler instead of the one of optee_os. So I would keep it as it is. This is not a string opinion, only my feeling. @jenswi-linaro just decides what you think is the best.